Automattic/jetpack
# Hoist Heartbeat into Connection.

You know the drill - Heartbeat now lives inside Connection, while the package itself is a dummy that depends on Connection to make sure all legacy packages work well.

#### Changes proposed in this Pull Request:
* The Heartbeat code is now included in Connection.
* The Heartbeat package is now abandoned.

#### Jetpack product discussion
* p9dueE-4Ep-p2

#### Does this pull request change what data or activity we track or use?
No
 
#### Testing instructions:





* Make sure the packages get build and assembled into plugins correctly.
* Make sure Heartbeat data is uploaded for new connections, as well as old ones.
